Makefile.PL

```diff
@@ -0,0 +1,79 @@
+#Makefile.PL version 1.0
+
+use 5.010001;
+use ExtUtils::MakeMaker;
+
+my @scripts   = qw(htseq2multx.pl);
+my $fqmx_link = "https://github.com/brwnj/fastq-multx";
+my $fqmx_name = 'fastq-multx';
+
+WriteMakefile
+  (
+   NAME            => 'htseq2multx',
+
+   #Add these new keywords supported since 5.005
+   ($[ >= 5.005 ?
+
+    (AUTHOR        => 'Robert W. Leach (rleach@princeton.edu)',
+     ABSTRACT_FROM => "$scripts[0]") : ()),
+
+   VERSION_FROM    => "$scripts[0]", # finds $VERSION
+   PREREQ_PM       => {'IO::Pipe::Producer'     => 2.01,
+                       'IO::Select'             => 1.21,
+                       'IO::Uncompress::Gunzip' => 2.060,
+                       'Getopt::Long'           => 2.38,
+                       'File::Glob'             => 1.17,
+                       'strict'                 => 1.07,
+                       'warnings'               => 1.13,
+                       'English'                => '1.06_01',
+                       'Readonly'               => 2.05},
+   EXE_FILES       => [@scripts]
+  );
+
+my $fastq_multx = getExe($fqmx_name);
+if(!defined($fastq_multx) || $fastq_multx eq '' ||
+   !-e $fastq_multx || !-x $fastq_multx)
+  {print STDERR ("WARNING: The $fqmx_name executable [$fastq_multx] appears ",
+                 'to either not be in your path, not exist, not have execute ',
+                 'permissions, or you have not created a symbolic link named ',
+                 "'$fqmx_name' to the full name of the executable with ",
+                 "version number.  If you have not installed $fqmx_name, ",
+                 "you can find it here: $fqmx_link\n")}
+
+
+
+
+
+sub getExe
+  {
+    my $command  = $_[0];
+    my $sent_exe = $command;
+    $sent_exe    =~ s/ .*//;
+    my $exe      = '';
+
+    if(eval("use File::Which;1;"))
+      {
+	$exe = which($sent_exe);
+	if((!defined($exe) || $exe eq '') && -e $sent_exe && -x $sent_exe)
+	  {$exe = $sent_exe}
+	elsif(!defined($exe))
+	  {$exe = ''}
```

Comment:
Suggest to convert tabs to spaces throughout the file.
```suggestion
  $exe = which($sent_exe);
  if((!defined($exe) || $exe eq '') && -e $sent_exe && -x $sent_exe)
    {$exe = $sent_exe}
  elsif(!defined($exe))
    {$exe = ''}
```

Reply:
Ugh. I must have a mis-config in xemacs.  It's set to not use tabs.  I'll have to figure that out.

- [ ] Remove tab characters

Reply:
Well, I figured out that [my no-tabs setting is a default that can be changed by various editing modules](https://emacs.stackexchange.com/questions/12173/confusion-by-the-usage-of-indent-tabs-mode-nil) (like the `cperl-mode` module I use, though I'm not sure that's the one that's doing it).  I explored the menus and set something that allowed me to replace the tabs with spaces (they were automatically changed back to tabs before I found that setting).  I'm not entirely sure though that the change I made will be permanent, but I'll revisit if I discover them again.